tests/unit/mocked/test_client_basic_behavior.py

```diff
@@ -0,0 +1,100 @@
+from image_go_nord_client.main import main
+from .unit_test_base_class import UnitTestBaseClass
+
+from unittest.mock import ANY
+
+
+class ClientShould(UnitTestBaseClass):
+    def test_return_docs_when_nothing_is_given(self):
+        with self.assertRaises(SystemExit):
+            main(argv=["image-go-nord-client"])
+
+        self.mock_gn_instance.open_image.assert_not_called()
+        self.mock_gn_instance.convert_image.assert_not_called()
+        self.assertIn(
```

Comment:
We should change the behaviour that return the -i/--img error when ign is called alone with the help page i think!

Reply:
Cool! I think this could be next improvement in a different issue